packages/components/src/DataList/components/DataListActions/DataListActions.tsx

```diff
@@ -27,6 +27,7 @@ export function DataListActions<T extends DataListObject>({
   return (
     <DataListOverflowFade>
       {exposedActions.map(({ props }) => {
+        if (props.visible && !props.visible(activeItem)) return null;
```

Comment:
> Mean you could get rid of the check for props.visible existing?

yeah, wrote it that way first but I also just want it to intentionally always show up unless otherwise stated not to.

> Also I think if icon is required [here](https://github.com/GetJobber/atlantis/pull/1626/files#diff-e48b68917ca593858e13de32ffc2f9ed3557c7a8572ae29e71fc120da08f9a7cR285) in the action prop definition the if statement below could be removed needed.

That's more of a TS precautionary shit. Either I start casting or just write it like it's not there as TS thinks it's still not there.

Reply:
Ahh fair. TS can be weird sometimes so that makes a ton of sense